src/strategies/DebtStrategy.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.16;
+
+import {ERC20} from "solmate/src/tokens/ERC20.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+import {SafeTransferLib} from "solmate/src/utils/SafeTransferLib.sol";
+
+import {AccessStrategy} from "src/strategies/AccessStrategy.sol";
+import {AffineVault} from "src/vaults/AffineVault.sol";
+
+contract DebtStrategy is AccessStrategy {
+    using SafeTransferLib for ERC20;
+
+    // amount of debt strategy has
+    uint256 public debt;
+
+    constructor(AffineVault _vault, address[] memory strategists) AccessStrategy(_vault, strategists) {}
+
+    /**
+     * @notice collect the funds from the investments
+     * @param amount amount needs to be withdraw
+     * @return amount withdrawn
+     */
+    function divest(uint256 amount) external override onlyVault returns (uint256) {
+        // increase the amount of debt
+        debt += amount;
```

Comment:
Only increment the debt if `divestedAmount` is less than amount. This is an unnecessary change to storage.